tokio-util/src/sync/cancellation_token/tree_node.rs

```diff
@@ -178,7 +178,7 @@ where
                 locked_node = node.inner.lock().unwrap();
                 locked_parent
             }
-            Err(TryLockError::Poisoned(err)) => Err(err).unwrap(),
+            Err(TryLockError::Poisoned(err)) => panic!("{err:?}"),
```

Comment:
Let's not make this change. I think calling into `unwrap` here generates less code.